lib/src/common/pipes/async_pipe.dart

```diff
@@ -39,22 +39,12 @@ var _observableStrategy = new ObservableStrategy();
 Future<dynamic> ___unused;
 
 /**
- * The `async` pipe subscribes to an Observable or Promise and returns the latest value it has
- * emitted.
- * When a new value is emitted, the `async` pipe marks the component to be checked for changes.
+ * An `async` pipe awaits for a value from a [Future] or [Stream]. When a value is received, 
+ * the `async` pipe marks the component to be checked for changes.
  *
  * ### Example
```

Comment:
I created this PR to fix some of it.
https://github.com/dart-lang/angular2/pull/29